src/GridFS/StreamWrapper.php

```diff
@@ -90,6 +93,30 @@ public static function register(string $protocol = 'gridfs'): void
         stream_wrapper_register($protocol, static::class, STREAM_IS_URL);
     }
 
+    /**
+     * Rename all revisions of a filename.
```

Comment:
I assume this only supports the bucket alias API, since we still only have a single `Bucket::rename()` method. Is this something that Drupal requires?

Reply:
I don't know if Drupal needs `rename`.